src/components/Homepage/LendByCategory/LoanCategoriesSection.vue

```diff
@@ -0,0 +1,341 @@
+<template>
+	<div>
+		<span
+			v-for="(category) in categories"
+			:key="category.id + '-link'"
+		>
+			<a
+				role="button"
+				@click.prevent="setActiveCategory(category.id)"
+				:class="{'active': category.id === activeCategory}"
+			>
+				{{ category.name }}
+			</a>
+		</span>
+		<div
+			class="loan-category-row"
+			v-for="(category, index) in categories"
+			v-show="category.id === activeCategory"
+			:key="category.id + '-row'"
+		>
+			<loan-category
+				:loan-channel="category"
+				:items-in-basket="itemsInBasket"
+				:row-number="index + 1"
+				:is-logged-in="isLoggedIn"
+				:is-visible="category.id === activeCategory"
+				ref="categoryRow"
+			/>
+		</div>
+		<div class="row pre-footer">
+			<div class="column small-12">
+				<div v-if="!rowLazyLoadComplete" class="cat-row-loader">
+					<loading-overlay id="updating-overlay" />
+					<h3 class="text-center">
+						Loading more rows...
+					</h3>
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import _drop from 'lodash/drop';
+import _each from 'lodash/each';
+import _get from 'lodash/get';
+import _map from 'lodash/map';
+import _take from 'lodash/take';
+import _uniqBy from 'lodash/uniqBy';
+// import cookieStore from '@/util/cookieStore';
+import logReadQueryError from '@/util/logReadQueryError';
+import { readJSONSetting } from '@/util/settingsUtils';
+import { indexIn } from '@/util/comparators';
+// import experimentQuery from '@/graphql/query/experimentAssignment.graphql';
+import lendByCategoryHomepageCategories from '@/graphql/query/lendByCategoryHomepageCategories.graphql';
+import loanChannelQuery from '@/graphql/query/loanChannelData.graphql';
+
+import LoanCategory from '@/components/Homepage/LendByCategory/LoanCategory';
+import LoadingOverlay from '@/pages/Lend/LoadingOverlay';
+
+// Row Limiter
+// > This controls how may rows are loaded on the server
+const ssrRowLimiter = 2;
+
+export default {
+	components: {
+		LoanCategory,
+		LoadingOverlay,
+	},
+	inject: ['apollo', 'kvAuth0'],
+	data() {
+		return {
+			categorySetting: [],
+			itemsInBasket: [],
+			prefetchedCategories: [],
+			clientCategories: [],
+			rowLazyLoadComplete: false,
+			rightArrowPosition: undefined,
+			leftArrowPosition: undefined,
+			isLoggedIn: false,
+			activeCategory: ''
+		};
+	},
+	apollo: {
+		query: lendByCategoryHomepageCategories,
+		preFetch(config, client) {
+			return client.query({
+				query: lendByCategoryHomepageCategories
+			}).then(({ data }) => {
+				// Get the array of channel objects from settings
+				const rowData = readJSONSetting(data, 'general.rows.value') || [];
+				// Get all channel ids for the row data
+				const ids = _map(rowData, 'id');
+
+				// Pre-fetch all the data for SSR targeted channels
+				return 	client.query({
```

Comment:
I don't think we should prefetch these so that we don't impact the initial load time too much. That's what I meant with R2.2